### PR TITLE
 update 0.1.13

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆Flatten_json 0.1.13 Update ☆ ❄️(Snowflake Pkg)
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2604)
[Upstream](https://github.com/amirziai/flatten/tree/v0.1.13)
[Dependencies ](https://github.com/amirziai/flatten/blob/v0.1.13/setup.py)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies

### CODE REVIEW

**Review by cbouss — APPROVED** [review]
LGTM, but abs.yaml should be added for upload to SF, right?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,29 +10,33 @@ source:
   sha256: ee352333e8293e957ccb1b4597a111fc4f6da88ab74b8cb3f8f51eed1e12f500
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Missing `entry_points`, see https://github.com/amirziai/flatten/blob/467934f7a8ecfdbc5a3100cfd042bea54140075a/setup.py#L16

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,29 +10,33 @@ source:
...
     - python
     - six
 
 test:
+  requires:
+    - pip
   imports:
     - flatten_json
+  commands:
+     - pip check
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:33]: Could you also add `flatten_json --help` because of `entry_points`?

**Review by rafaelmartins — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:33]: @Arishamays1 Hm, something is wrong with `flatten_json --help`. Could you check it locally?

**Comment by lorepirri** [comment]
This seems like a Snowflake package. @Arishamays1 could you please follow [these guidelines](https://github.com/anaconda-distribution/perseverance-skills/blob/building-snowflake-packages/sections/02_Package_building/01_How_tos/Building_Snowflake_packages.md)? They are still under review but almost complete. Please ping me if you encounter any errors or unclear steps.

File: recipe/meta.yaml (hunk 2 above)
Comment (Arishamays1) [reply recipe/meta.yaml:33]: Added, Thanks for the suggestion! It seems this package gets stuck on this test command when adding this to the meta.yaml both here onPrefect and locally. I will remove this as I do not believe it is a working feature.

**Comment by Arishamays1** [comment]
> This seems like a Snowflake package. @Arishamays1 could you please follow [these guidelines](https://github.com/anaconda-distribution/perseverance-skills/blob/building-snowflake-packages/sections/02_Package_building/01_How_tos/Building_Snowflake_packages.md)? They are still under review but almost complete. Please ping me if you encounter any errors or unclear steps.

Of course! will do!

**Pull request merged**